web/src/validators/social-media-urls.js

```diff
@@ -0,0 +1,18 @@
+import { helpers } from 'vuelidate/lib/validators';
+
+const linkedinUrl = (value) => {
+  const regex = /^https?:\/\/((www|\w\w)\.)?linkedin.com\/((in\/[^/]+\/?)|(pub\/[^/]+\/((\w|\d)+\/?){3}))$/;
+  return !helpers.req(value) || regex.test(value);
+};
+
+const facebookUrl = (value) => {
```

Comment:
Ensure we allow a trailing slash here:

<img width="495" alt="SpeakHer" src="https://user-images.githubusercontent.com/4602369/122675904-93758c80-d216-11eb-882b-caeab77c361b.png">

Reply:
@ann-kilzer It should support trailing dash here